mango-core/src/main/java/org/calrissian/mango/domain/BaseAttributeStore.java

```diff
@@ -31,22 +31,21 @@
  */
 public class BaseAttributeStore implements AttributeStore {
 
-    private Multimap<String, Attribute> attributes = ArrayListMultimap.create();
+    protected Multimap<String, Attribute> attributes;
```

Comment:
If you are going to make this protected then, in the constructor wrap in with an unmodifiableMultiMap so that it can't be mutated.